src/backend/ingestion/index.ts

```diff
@@ -88,6 +96,7 @@ export class Ingestion extends Construct implements IGrantable {
         STATE_MACHINE_ARN: props.orchestration.stateMachine.stateMachineArn,
         PACKAGE_LINKS: JSON.stringify(props.packageLinks ?? []),
       },
+      logRetention: props.logRetention,
```

Comment:
Maybe make this default explicit in case the AWS defaults change under our feet and our docs become outdated? This would also need to be updated in `construct-hub.ts`
```suggestion
      logRetention: props.logRetention ?? RetentionDays.TEN_YEARS,
```